packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -552,6 +552,7 @@ void main() {
         'LICENSE',
         'README.md',
         'example/lib/main.dart',
+        'example/integration_test/plugin_integration_test.dart',
```

Comment:
Can we add an explicit test here to create an integration test project?

Reply:
Sure, done.